src/integration/resources/types.go

```diff
@@ -66,6 +66,9 @@ type Coordinator interface {
 	// WritePromWithLabels writes a prometheus metric. Allows you to provide the labels for
 	// the write directly instead of conveniently converting them from a map.
 	WritePromWithLabels(name string, labels []prompb.Label, samples []prompb.Sample, headers Headers) error
```

Comment:
nit: With this new method, we don't really need `WritePromWithLabels` anymore. Would be nice if we delete this and replace the one reference that currently exists.